docs/language-usage.rst

```diff
@@ -0,0 +1,354 @@
+                     'beginOffset': -1
+                 },
+                 'dependencyEdge': {
+                     'headTokenIndex': 1,
+                     'label': 'DET'
+                 },
+                 'partOfSpeech': {
+                     'tag': 'DET'
+                 },
+                 'lemma': 'The'
+             },
+             {
+                 'text': {
+                     'content': 'cow',
+                     'beginOffset': -1
+                 },
+                 'dependencyEdge': {
+                     'headTokenIndex': 2,
+                     'label': 'NSUBJ'
+                 },
+                 'partOfSpeech': {
+                     'tag': 'NOUN'
+                 },
+                 'lemma': 'cow'
+             },
+             {
+                 'text': {
+                     'content': 'jumped',
+                     'beginOffset': -1
+                 },
+                 'dependencyEdge': {
+                     'headTokenIndex': 2,
+                     'label': 'ROOT'
+                 },
+                 'partOfSpeech': {
+                     'tag': 'VERB'
+                 },
+                 'lemma': 'jump'
+             },
+             {
+                 'text': {
+                     'content': 'over',
+                     'beginOffset': -1
+                 },
+                 'dependencyEdge': {
+                     'headTokenIndex': 2,
+                     'label': 'PREP'
+                 },
+                 'partOfSpeech': {
+                     'tag': 'ADP'
+                 },
+                 'lemma': 'over'
+             },
+             {
+                 'text': {
+                     'content': 'the',
+                     'beginOffset': -1
+                 },
+                 'dependencyEdge': {
+                     'headTokenIndex': 5,
+                     'label': 'DET'
+                 },
+                 'partOfSpeech': {
+                     'tag': 'DET'
+                 },
+                 'lemma': 'the'
+             },
+             {
+                 'text': {
+                     'content': 'moon',
+                     'beginOffset': -1
+                 },
+                 'dependencyEdge': {
+                     'headTokenIndex': 3,
+                     'label': 'POBJ'
+                 },
+                 'partOfSpeech': {
+                     'tag': 'NOUN'
+                 },
+                 'lemma': 'moon'
+             },
+             {
+                 'text': {
+                     'content': '.',
+                     'beginOffset': -1
+                 },
+                 'dependencyEdge': {
+                     'headTokenIndex': 2,
+                     'label': 'P'
+                 },
+                 'partOfSpeech': {
+                     'tag': 'PUNCT'
+                 },
+                 'lemma': '.'
+             }
+         ],
+         'entities': [],
+         'documentSentiment': {
```

Comment:
This is weird. Not sure why we have `language`, `tokens`, `entities` and then `documentSentiment` instead of just `sentiment`...